src/language-handlebars/utils.js

```diff
@@ -41,6 +41,7 @@ const voidTags = new Set(htmlVoidElements);
 function isVoid(node) {
   return (
     voidTags.has(node.tag) ||
+    node.selfClosing === true ||
```

Comment:
Should we check this in print? So we can keep `<i></i>`?

Reply:
Do you think `<i/>` should be formatted to `<i></i>`? While `<div />` should remain `<div />`? (inline elements vs others)

(Just pushed more tests and verbose changelog to better describe the changes.)

Reply:
No, I mean maybe we should keep it as it is. It's not our job to enforce the self close style.

Reply:
`<i></i>` -> `<i></i>` 
`<i/>`  -> `<i/>`

Reply:
oh I see, thanks for the clarification
I have added more test cases in the PR to show it doesn't enforce the self close style (except for the `<Component />` case, but that was already the case prior to this PR, for a reason I don't remember)

Reply:
Ah, sorry, I misunderstood.